chart/templates/cdn-invalidator-sub/deployment.yaml

```diff
@@ -182,6 +187,11 @@ spec:
             items:
```

Comment:
Wait a second, the cdn-invalidator-sub haven't access to redis? :thinking:

Reply:
Does it need it?

Reply:
I don't know

Reply:
Confirmed with Tarrero that is not necessary.